packages/compass-connections-navigation/src/connection-item.tsx

```diff
@@ -101,14 +101,40 @@ export const ConnectionItem: React.FunctionComponent<
   );
 
   const actions: ItemAction<Actions>[] = useMemo(() => {
-    return [
+    const isFavorite = connectionInfo.savedConnectionType === 'favorite';
+
+    const actions: ItemAction<Actions>[] = [
+      {
+        action: 'connection-performance-metrics',
+        icon: 'Gauge',
+        label: 'View performance metrics',
+      },
+      {
+        action: 'open-connection-info',
+        icon: 'InfoWithCircle',
+        label: 'Show connection info',
+      },
       {
-        action: 'connection-options', // TODO(COMPASS-7714)
-        icon: 'Ellipsis',
-        label: 'Options',
+        action: 'copy-connection-string',
+        icon: 'Copy',
+        label: 'Copy connection string',
+      },
+      {
+        action: 'connection-toggle-favorite',
+        icon: 'Favorite',
+        label: isFavorite ? 'Unfavorite' : 'Favorite',
       },
     ];
-  }, []);
+
+    actions.push({
+      action: 'connection-disconnect',
+      icon: 'Disconnect',
+      label: 'Disconnect',
+      variant: 'destructive',
+    });
```

Comment:
This can just be part of the array above, right?

Reply:
ah, yes, forgot to merge this. thanks!